.github/workflows/build.yml

```diff
@@ -58,17 +57,17 @@ jobs:
       #  with:
       #    name: ccache-debug.tar.gz
       #    path: ccache-debug.tar.gz
-      - name: upload sandstorm tarball
-        uses: actions/upload-artifact@v1
-        with:
-          name: sandstorm-${{ env.BUILD }}-fast.tar.xz
-          path: sandstorm-${{ env.BUILD }}-fast.tar.xz
+      #- name: upload sandstorm tarball
+      #  uses: actions/upload-artifact@v1
+      #  with:
+      #    name: sandstorm-${{ env.BUILD }}-fast.tar.xz
```

Comment:
Might as well replace `${{ env.BUILD }}` here with `0`, to avoid confusing anyone who wants to uncomment this...

Reply:
I agree. Am tempted to ask you to "squash and merge" this ugly as heck PR when I'm done with it. But hey, it worked this time!